### DISCUSSION TITLE
Instal fresh on nas (qnap)

### DISCUSSION THREAD

**Discussion by lelbo** [LycheeOrg/Lychee#2280] [Q&A] [answered]
Hello 
I have try to install lytchee.
but no connect mariaDB.  
No table, and same is me  create database..
I try With admin account.

After first install ... I m redirect on /galléries but no login and only messages.. 
You have no public folder....

I am on nas Qnap .. no compose no docker. Only gît instal 

Other CMS work 

Tank

**Comment by ildyria** [comment, marked as answer]
Use the zip release.
https://github.com/LycheeOrg/Lychee/releases/latest

Then check the nas-ip:port/xxx/Diagnostics and see what is broken.
That is pretty much all we can do to help you.

If I were you, I would see how to get docker running on Qnap ( https://nasbuddy.com/docker-installation-guide-for-qnap-nas/) .
